examples/auth-provider.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const pogobuf = require('../pogobuf/pogobuf.js');
+
+//login types are ptc and google
+var client = new pogobuf.Client('loginType','username','password');
+
+client.init()
+    .then(function(data){
+        console.log('init success!');
```

Comment:
Honestly, in libraries I prefer the pattern of allowing the user to passing in a logging object, or defaulting to `console.log`. However, that's a larger, unrelated change, and probably should be brought up in an issue. I'd just remove the logging from this merge.